hypex/ab_test/limit_distribution.py

```diff
@@ -61,53 +62,45 @@ def quantile_of_marginal_distribution(
         num_samples: The number of samples/groups to compare.
         quantile_level: The quantile level to compute for the marginal distribution (e.g., 0.95 for the 95th percentile).
         variances: A list of variances for each sample/group. If None, equal variances are assumed.
+        equal_variance: A boolean indicating if the samples are assumed to have equal variance (default is True).
         iteration_size: The number of iterations/random samples to generate for the simulation.
-        random_state: Optional parameter for data randomisation.
+        random_state: Random state. (Default is 42)
```

Comment:
It is better to set the default value to None, so that there is no fixed random state.